src/api/instance-methods.md

```diff
@@ -32,7 +36,17 @@
         // do something
       })
 
-      // function
+      // function for watching a deep path
```

Comment:
**suggestion**: let's replace `deep path` with `nested property` to reduce the ambiguity with `deep` parameter
```suggestion
      // function for watching a single nested property
```